single_tech_samples/synapseanalytics/sample2_synapseserverless/scripts/deploy_synapse_artifacts.sh

```diff
@@ -143,6 +143,25 @@ do
     echo "$provision_state: checking again in 10 seconds..."
 done
 
+addPackageToSynapseWorkspace(){
+    echo "$(date):Synapse Workspace: Adding Wheel file as a workspace package"
+    az synapse workspace-package upload --workspace-name "${SYNAPSE_WORKSPACE_NAME}" \
+        --package ./synapseartifacts/workspace/workspace_packages/adlsaccess-1.0-py3-none-any.whl
+}
+
+attachingPackageToSparkPool(){
+    echo "$(date):Synapse SparkPool Update: Attaching workspace package to SparkPool"
+    az synapse spark pool update --name "${BIG_DATAPOOL_NAME}" \
+        --workspace-name "${SYNAPSE_WORKSPACE_NAME}" \
+        --resource-group "${RESOURCE_GROUP_NAME}" \
+        --package "adlsaccess-1.0-py3-none-any.whl" \
+        --package-action Add
+}
+
+# Add Whell package to Synapse workspace
+addPackageToSynapseWorkspace
+attachingPackageToSparkPool
```

Comment:
in case you rename to attach instead of attaching, this also needs to be fixed

Reply:
Corrected